src/DIRAC/FrameworkSystem/DB/ProxyDB.py

```diff
@@ -1347,134 +1334,76 @@ def useToken(self, token, requesterDN, requesterGroup):
-                        lTime,
-                    )
-                    cmd = (
-                        "INSERT INTO `ProxyDB_ExpNotifs` ( UserDN, UserGroup, LifeLimit, ExpirationTime ) VALUES %s"
-                        % values
-                    )
-                    result = self._update(cmd)
-                    if not result["OK"]:
-                        gLogger.error("Could not mark notification as sent", result["Message"])
-                else:
-                    values = "LifeLimit = %d, ExpirationTime = TIMESTAMPADD( SECOND, %s, UTC_TIMESTAMP() )" % (
-                        notifLimit,
-                        lTime,
-                    )
-                    cmd = "UPDATE `ProxyDB_ExpNotifs` SET {} WHERE UserDN = {} AND UserGroup = {}".format(
-                        values,
-                        sUserDN,
-                        sGroup,
-                    )
-                    result = self._update(cmd)
-                    if not result["OK"]:
-                        gLogger.error("Could not mark notification as sent", result["Message"])
-                sent.append((userDN, group, lTime))
-                notifDone[notKey] = notifLimit
-        return S_OK(sent)
-
-    def _notifyProxyAboutToExpire(self, userDN, lTime):
+
+        return S_OK()
+
+    def _notifyProxyAboutToExpire(self, userName, lTime):
         """Send notification mail about to expire
 
-        :param str userDN: user DN
-        :param int lTime: left proxy live time in a seconds
+        :param str userName: user name
+        :param int lTime: left proxy live time in seconds
 
         :return: boolean
         """
-        result = Registry.getUsernameForDN(userDN)
-        if not result["OK"]:
-            return False
-        userName = result["Value"]
-        userEMail = Registry.getUserOption(userName, "Email", "")
-        if not userEMail:
+        if not (userEMail := Registry.getUserOption(userName, "Email")):
             gLogger.error("Could not discover user email", userName)
             return False
+        if not (userDN := Registry.getUserOption(userName, "DN")):
+            gLogger.error("Could not discover user DN", userName)
+            return False
         daysLeft = int(lTime / 86400)
         if daysLeft <= 0:
             return True  # Don't annoy users with -1 messages
         msgSubject = "Your proxy uploaded to DIRAC will expire in %d days" % daysLeft
-        msgBody = """\
-Dear %s,
+        msgBody = textwrap.dedent(
+            """\
+                Dear %s,
+
+                The proxy you uploaded to DIRAC will expire in aproximately %d days. The proxy
+                information is:
+
+                DN: %s
 
-  The proxy you uploaded to DIRAC will expire in aproximately %d days. The proxy
-  information is:
+                If you plan on keep using this credentials, please upload a newer proxy by executing:
 
-  DN:    %s
+                $ dirac-proxy-init
```

Comment:
Shall we make this configurable so we can chage it in the LHCbDIRAC CS?

Reply:
`lhcb-proxy-init` is there for historical reasons, not technical anymore (I know, you can run it without `lb-dirac` but do we need really that?)

Reply:
I think we should keep `lhcb-proxy-init` as an executable, but I wouldn't bother making an option in the CS just for that mail